circle.yml

```diff
@@ -11,6 +14,14 @@ machine:
     LOG_LEVEL: 40
     ADMINS: '["thedude@example.com"]'
     HMAC_KEYS: '{"2013-01-01": "prositneujahr"}'
+    # Deis
+    DEIS_CONTROLLER: https://deis.us-west.moz.works
+    DEIS_USERNAME: circleci
```

Comment:
Deis user is created but I've not yet updated the `DEIS_PASSWORD` variable in circleci itself.